keras_hub/src/tests/test_case.py

```diff
@@ -569,6 +569,14 @@ def run_task_test(
         ds = tf.data.Dataset.from_tensor_slices(train_data).batch(batch_size)
         x, y, sw = keras.utils.unpack_x_y_sample_weight(train_data)
 
+        # Test: shapes output by the preprocessor must match what model expects.
```

Comment:
This is not just checking the shape, it's checking the entire structure, right?

Reply:
yes, I clarified the comment